src/test/java/uk/gov/hmcts/reform/bulkscanprocessor/model/mapper/SubtypeTest.java

```diff
@@ -51,13 +47,15 @@ public void should_map_scannable_item_document_types_correctly() {
             ScannableItem result = EnvelopeMapper.toDbScannableItem(item);
 
             // then
-            softly.assertThat(result.getDocumentType())
+            assertSoftly(softly -> softly.assertThat(result.getDocumentType())
                 .as("Output document type for type '%s' and subtype '%s'", tc.input.documentType, tc.input.docSubtype)
-                .isEqualTo(tc.expected.documentType);
+                .isEqualTo(tc.expected.documentType)
+            );
 
-            softly.assertThat(result.getDocumentSubtype())
+            assertSoftly(softly -> softly.assertThat(result.getDocumentSubtype())
```

Comment:
if you decide to go this way than all assertions should go into on `assertSoftly`

Reply:
alternatively
`SoftAssertions softly = new SoftAssertions();` before assertions
and
`softly.assertAll()` after assertions